src/components/TodoItem.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+const TodoItemContainer = styled.div`
+  padding: 3px 1.5rem 3px 2rem;
+  display: flex;
+  justify-content: space-between;
+
+  //toggle 스타일링
+  text-decoration: ${(props) => (props.isCompleted ? 'line-through' : ' ')};
```

Comment:
`none`이 더 좋을 것 같습니다
또한 destructuring을 이용하면 더 짧게 쓸 수 있겠죠?

```suggestion
  text-decoration: ${({ isCompleted }) => (isCompleted ? 'line-through' : 'none')};
```